### ISSUE TITLE
2.8.0-beta2 pre-release plan

### ISSUE THREAD

**Issue by joeparsons** [az-digital/az_quickstart#2832]
- #2808 
- #2830 
- #2826
- #2821 
- #2822 
- #2828
- #2833

**Comment by joeparsons** [comment]
## Pick list

```
git cherry-pick db614253a6195ec969ad357c04062872cf2d61d3
git cherry-pick abcd881a940961b40edddf64ca8a10b36b5a2ccc
git cherry-pick 9f998833c51384634157b1ff3761f7c68db52370
git cherry-pick 6585288def7b0349788529e5c2c5331e333d80ce
git cherry-pick 0b0ce740199112ed8af5c571a6fb0047e7bba1ff
git cherry-pick 74ec70776fc0d7f13daad2183375de18704a7fa1
git cherry-pick 1966e8019b8cd0ed520d944951dcf16a80c658eb
```

**Comment by joeparsons** [comment]
## Release notes snippet

`git log 2.8.0-beta1..HEAD --pretty='format:- %s'`

```
_This is a pre-release for the next minor version (feature release) of Arizona Quickstart 2.  Pre-releases are not supported for production sites, but they are intended for widespread testing in preparation for the upcoming stable release._

- Closes #2829 All external links opening in new tab instead of only intended external links (#2833)
- Update drupal/blazy requirement from 2.19.0 to 2.20.0 (#2828)
- Update drupal/pathauto requirement from 1.11.0 to 1.12.0 (#2822)
- Update drupal/webform requirement from 6.2.0-beta7 to 6.2.0-rc1 (#2821)
- Closes #2824 Visual changes to node edit form with 2.8 pre-release (#2826)
- Closes #2825, Closes #2827 Add module installer for views_remote_data in update hook. (#2830)
- Update drupal/linkit requirement from 6.1.1 to 6.1.2 (#2808)
- Back to dev.
```

**Comment by joeparsons** [comment]
Release created: https://github.com/az-digital/az_quickstart/releases/tag/2.8.0-beta2
